### PR TITLE
Format new code with clang-format

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

File: src/api/wordstrboxrenderer.cpp [hunk 1]
```diff
@@ -44,31 +44,32 @@ char* TessBaseAPI::GetWordStrBoxText(int page_number) {
...
-        res_it->Next(RIL_WORD);
-     } while (!res_it->Empty(RIL_BLOCK) && !res_it->IsAtBeginningOf(RIL_WORD));
+    do {
+      wordstr_box_str +=
+          std::unique_ptr<const char[]>(res_it->GetUTF8Text(RIL_WORD)).get();
+      wordstr_box_str += " ";
+      res_it->Next(RIL_WORD);
+    } while (!res_it->Empty(RIL_BLOCK) && !res_it->IsAtBeginningOf(RIL_WORD));
   }
   wordstr_box_str.add_str_int("\n\t ", right + 1);
```
Comment (stweil) [inline src/api/wordstrboxrenderer.cpp]: @Shreeshrii, `right`, `top` and `bottom` can be uninitialized here (reported by Coverity Scan). What would be the right fix for this?

**Comment by amitdo** [comment]
>Format new code with clang-format

What about old code?

**Comment by stweil** [comment]
`src/api/wordstrboxrenderer.cpp` was added recently, so there is no old code.

**Comment by amitdo** [comment]
I meant what about other files?

File: src/api/wordstrboxrenderer.cpp (hunk 1 above)
Comment (stweil) [reply src/api/wordstrboxrenderer.cpp]: There is a similar problem in [lstmboxrenderer.cpp:82](https://github.com/tesseract-ocr/tesseract/blob/master/src/api/lstmboxrenderer.cpp#L82) and 3 more lines.

**Comment by stweil** [comment]
The new code got my attention because of the [latest report from Coverity Scan](https://scan.coverity.com/projects/tesseract-ocr?tab=overview).

But you are right, the old code needs formatting, too.

**Comment by zdenop** [comment]
@stweil: do you plan to fix issues from Coverity Scan?

**Comment by stweil** [comment]
Not at the moment. I am still waiting for feedback from @Shreeshrii (see question above).

File: src/api/wordstrboxrenderer.cpp (hunk 1 above)
Comment (Shreeshrii) [reply src/api/wordstrboxrenderer.cpp]: @stweil For some reason I did not get any email regarding the comments on the commits. Got the msg re issue today.

The EOL tab needs to be added at end of a line. Currently it is identified via IsAtBeginningOf(RIL_TEXTLINE). However that would put an EOL tab box at the beginning of the page,  hence I am checking that it is NOT the first line on page.

There is probably a different way to structure the code to get rid of this problem.  Please change the code as you see fit. Thanks!

**Comment by Shreeshrii** [comment]
@stweil I hadn't recd the earlier msg with the review. Please feel free to change the code. Thanks.

**Pull request merged**